embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/arch/EmbraceAttributeExtensions.kt

```diff
@@ -83,12 +83,12 @@ internal fun Span.assertDoesNotHaveEmbraceAttribute(embraceAttribute: EmbraceAtt
 }
 
 internal fun Span.assertError(errorCode: ErrorCode) {
-    assertEquals(StatusCode.ERROR, status)
+    assertEquals(Span.Status.ERROR, status)
```

Comment:
This was a bug that wasn't found because the code path of the existing tests never got here. Until now :-)